packages/core/src/events/TapEvent.ts

```diff
@@ -1,7 +1,5 @@
 import { global } from '../utils/global.js';
 
-const isIE = (/Trident/g).test(navigator.userAgent);
```

Comment:
We should find the same code by searching keyword `Trident` in the repository and remove all.